.github/workflows/torizon_demos.yaml

```
@@ -21,23 +21,40 @@ jobs:
         strategy:
             matrix:
                 include:
-# Disabled armhf, as armv7 seems to have vanished from https://hub.docker.com/r/torizon/wayland-base-vivante/tags
-#                    - target: armhf
-#                      image_arch: linux/arm/v7
-#                      rust_toolchain_arch: armv7-unknown-linux-gnueabihf
-#                      base_image_suffix: ""
+                    # Default software rendering (GPU-less)
                     - target: arm64
                       image_arch: linux/arm64
                       rust_toolchain_arch: aarch64-unknown-linux-gnu
                       base_image_suffix: ""
-#                    - target: armhf
-#                      image_arch: linux/arm/v7
-#                      rust_toolchain_arch: armv7-unknown-linux-gnueabihf
-#                      base_image_suffix: "-vivante"
+                      slint_backend_type: "linuxkms-software"
```

I'd like to suggest to default to the Skia software renderer for these rather powerful devices.

```suggestion
                      slint_backend_type: "linuxkms-skia-software"
```

That said, in principle I think it would be cleanest to merely set

`SLINT_BACKEND=linuxkms`

and let Slint determine the correct renderer at run-time. I'm saying this because the fallback to (Skia) software when no GPU is present should work, and because we'll want to switch to Vulkan in the future if available (via wgpu). It would be nice if these wouldn't require changes in the docker files here.